### PR TITLE
Added html for header

### LINKED ISSUE #11: html header
We need to create the layout of the header

### PR DESCRIPTION
**Pull request by Talina24**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,49 @@
...
+        <select class="header__lang" id="lang">
+          <option value="en">EN</option>
+          <option value="ru">RU</option>
+        </select>
+        <!-- Theme-->
+        <button class="header__theme" type="button" id="theme">
+          <img src="assets/icons/moon.svg" alt="Moon" />
+        </button>
+        <!-- Burger-->
+        <button class="header__burger" type="button" id="burger">
```
Comment (ghost) [inline index.html:48]: Add tag span
```
<span></span>
```

**Review by ghost — COMMENTED** [review]
Fix

File: index.html (hunk 1 above)
Comment (Talina24) [reply index.html:48]: fixed

**Review by ghost — APPROVED** [review]
Done

**Pull request merged**